web_development_101/the_front_end/project_html_css.md

```diff
@@ -94,6 +94,7 @@ There are a couple of ways to go about doing this, but the simplest is this:
 Submit a pull request with a link to your solution in this [file](https://github.com/TheOdinProject/curriculum/blob/master/web_development_101/the_front_end/project_html_css.md) when your project is complete. See the section on [Contributing](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md) for how.
 
 * Add your solution below this list
+*[Michael Lorenzana Solution (Easy)](ttps://mlorenzana12.github.io/google_homepage/) - [View in Browser](https://mlorenzana12.github.io/google_homepage/)
```

Comment:
Please fix the formatting for the solution you provided.

There should be a space after the `*` in order for the list-item to render correctly in markdown.
The first link you provide should also be a URL your github repository for this project.